Gemfile.lock

```diff
@@ -392,3 +384,6 @@ DEPENDENCIES
   webmock (= 1.17.3)
   whenever (~> 0.9.4)
   zeroclipboard-rails
+
+BUNDLED WITH
+   1.10.6
```

Comment:
This shouldn't be present for a ruby 2.2.2 app (we're running bundler 1.9.4 with ruby 2.2.2).